cypress/e2e/gui/engageSphere.cy.js

```diff
@@ -0,0 +1,13 @@
+describe('Testes dos componentes da aplicação Engage Sphere', () => {
+  beforeEach(() => {
+    cy.visit('/')
+  })
```

Comment:
Para ajudar na legibilidade, recomendo separar o `beforeEach` do `it` por uma linha em branco.

> O mesmo vale para a separação de um `it` do outro.

```suggestion
  })
  
```